src/Bridges/Constraint/chance.jl

```diff
@@ -0,0 +1,171 @@
+    ::Type{<:MOI.AbstractVectorFunction},
+    ::Type{MOI.IndependentNormalChance{T, S}}) where {T, S<:LessOrGreater{T}}
+    return true
+end
+MOIB.added_constrained_variable_types(::Type{<:ChancetoSOCBridge}) = Tuple{DataType}[]
+function MOIB.added_constraint_types(::Type{<:ChancetoSOCBridge{T, F}}) where {T, F}
+    return [(F, MOI.SecondOrderCone)]
+end
+
+function concrete_bridge_type(::Type{<:ChancetoSOCBridge{T}},
+                              G::Type{<:MOI.AbstractVectorFunction},
+                              ::Type{MOI.IndependentNormalChance{T, S}}) where T
+    S = MOIU.promote_operation(+, G, G)
+    F = MOIU.promote_operation(*, T, S)
+    return ChancetoSOCBridge{T, F, S, G}
+end
+
+# Attributes, Bridge acting as a model
+function MOI.get(::ChancetoSOCBridge{T, F},
+                 ::MOI.NumberOfConstraints{F, MOI.SecondOrderCone}) where {T, F}
+    return 1
+end
+function MOI.get(bridge::ChancetoSOCBridge{T, F},
+                 ::MOI.ListOfConstraintIndices{F, MOI.SecondOrderCone}) where {T, F}
+    return [bridge.constraint]
+end
+
+# References
+function MOI.delete(model::MOI.ModelLike, bridge::ChancetoSOCBridge)
+    MOI.delete(model, bridge.constraint)
+end
+
+# Attributes, Bridge acting as a constraint
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::Union{MOI.ConstraintPrimalStart, MOI.ConstraintDualStart},
+    ::Type{<:ChancetoSOCBridge})
+
+    return true
+end
+function MOI.get(model::MOI.ModelLike,
+                 attr::Union{MOI.ConstraintPrimal, MOI.ConstraintPrimalStart},
+                 bridge::ChancetoSOCBridge)
+    return _inverse_map_function(
+        typeof(bridge), MOI.get(model, attr, bridge.constraint),
+        bridge.set,
+        attr isa MOI.ConstraintPrimal &&
+            MOIU.is_ray(MOI.get(model, MOI.PrimalStatus(attr.N))))
+end
+function MOI.set(model::MOI.ModelLike, attr::MOI.ConstraintPrimalStart,
+                 bridge::ChancetoSOCBridge, value)
+    MOI.set(model, attr, bridge.constraint,
+            _map_function(typeof(bridge), value, bridge.set, false))
+end
+function MOI.get(model::MOI.ModelLike,
+                 attr::Union{MOI.ConstraintDual, MOI.ConstraintDualStart},
+                 bridge::ChancetoSOCBridge)
+    return _adjoint_map_function(
+        typeof(bridge), MOI.get(model, attr, bridge.constraint),
+        bridge.set)
+end
+function MOI.set(model::MOI.ModelLike, attr::MOI.ConstraintDualStart,
+                 bridge::ChancetoSOCBridge, value)
+    MOI.set(model, attr, bridge.constraint, _inverse_adjoint_map_function(
+        typeof(bridge), MOI.get(model, attr, bridge.constraint),
+        bridge.set))
+end
+function MOI.modify(model::MOI.ModelLike, bridge::SetMapBridge,
+                    change::MOI.VectorConstantChange)
+    # By linearity of the map, we can just change the constant
+    constant = _map_function(typeof(bridge), change.new_constant, bridge.set, true)
+    MOI.modify(model, bridge.constraint, MOI.VectorConstantChange(constant))
```

Comment:
Missing `return` here and above (http://www.juliaopt.org/JuMP.jl/dev/style/#Return-statements-1).